app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -455,11 +476,11 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
 
     override fun success(route: Route) {
         routeManager?.route = route
-        routePreviewView?.route = route
```

Comment:
I like the cleanup removing the optional checking here. However, ignorant Android question - findViewById will always return the correct type or crash, yes?